.codeclimate.yml

```diff
@@ -51,3 +51,4 @@ ratings:
   - "**.py"
 exclude_paths:
 - "adage/**/migrations/"
+- "interface/**/*-vgspec.js"
```

Comment:
Right, and it seems to have worked. Codeclimate had nothing more to say about `volcano-plot-vgspec.js` after I moved the exclusion here.